consensus/ising/voting/sigchain.go

```diff
@@ -95,7 +95,7 @@ func (scv *SigChainVoting) SetVotingHeight(height uint32) {
 }
 
 func (scv *SigChainVoting) GetVotingHeight() uint32 {
-	return ledger.DefaultLedger.Store.GetHeight() + 2
+	return ledger.DefaultLedger.Store.GetHeight() + 3
```

Comment:
No magic number, use const name or explain why 2&3 be used here

Reply:
Fixed.